spec/features/creating_an_authorised_fuel_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "spec_helper"
```

Comment:
These tests are admirable 👍 I feel bad in saying I hope they can be [deleted very shortly](https://trello.com/c/VfGijIuK/765-dry-up-the-specialist-publisher-tests)...!